jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/resources/slack-soy/channelmapping/channelmapping.soy

```diff
@@ -135,13 +135,16 @@
                     </tr>
                   </thead>
                 {foreach $projectConfiguration in $projectConfigurations}
-                    {foreach $channelId in $projectConfiguration.orderedChannelIds}
-                        {foreach $configTeam in $projectConfiguration.configuration[$channelId]}
+                    {foreach $conversationKey in $projectConfiguration.orderedChannelIds} //ConversationKey
+                    {let $convKey: $conversationKey.teamId + ':' + $conversationKey.channelId /}
+                        {foreach $configTeam in $projectConfiguration.configuration[$convKey]} //List<ConfigurationGroupDTO>
+                        <!-- settings $configTeam.settings  -->
+                        <!-- config group id $configTeam.configurationGroupId -->
```

Comment:
I suppose these comments that were used for local debugging aren't needed anymore and need to be removed.